bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -261,12 +266,11 @@ pub mod pallet {
 					Error::<T>::from(error)
 				})?;
 
-			// Pay relayer reward
-			T::RewardPayment::register_reward(
-				&relayer,
-				T::DefaultRewardKind::get(),
-				message.relayer_fee,
-			);
+			// Get the tip and remove it from storage, if a tip was added.
+			let tip_amount = Tips::<T>::take(message.nonce);
```

Comment:
Just naming this `tip` is fine.

```suggestion
			let tip = Tips::<T>::take(message.nonce);
```

Reply:
Addressed in https://github.com/vgeddes/polkadot-sdk/pull/23/commits/52404b229def2f307f44f552b61f3000adaf9cac.